frontend/src/api/dateHandling.ts

```diff
@@ -0,0 +1,58 @@
+export function getMondayAndFridayDates(): { currentMonday: string, currentFriday: string} {
+    const now = new Date();
+    const currentDayOfWeek = now.getDay(); // 0 = Sunday, 1 = Monday, ..., 6 = Saturday
+  
+    // Calculate the date of the Monday of the current week
+    const monday = new Date(now);
+    monday.setDate(now.getDate() - currentDayOfWeek + 1);
+  
+    // Calculate the date of the Friday of the current week
+    const friday = new Date(monday);
+    friday.setDate(monday.getDate() + 4);
+  
+    // Format the dates as strings in the format YYYYMMDD
+    const currentMonday = monday.toISOString().slice(0, 10).replace(/-/g, '');
+    const currentFriday = friday.toISOString().slice(0, 10).replace(/-/g, '');
+  
+    // Return an object with the Monday and Friday dates
+    return { currentMonday, currentFriday };
+}
+export function shiftForward(monday: string, friday: string): { currentMonday: string, currentFriday: string } {
+    const mondayDate: Date = new Date(Number(monday.slice(0, 4)), Number(monday.slice(4, 6)) - 1, Number(monday.slice(6)));
```

Comment:
make a function for `new Date(Number(days.slice(0,4)), Number(day.slice(4,6)) -1` since this is always the same task and repeats quite a lot

Reply:
Oh and while you're at it, comment it, this is unreadable